amm/repl/src/main/scala/ammonite/main/Defaults.scala

```diff
@@ -25,32 +25,37 @@ object Defaults{
   // Need to import stuff from ammonite.ops manually, rather than from the
   // ammonite.ops.Extensions bundle, because otherwise they result in ambiguous
   // imports if someone else imports maunally
-  val predefString = s"""
-    |import ammonite.ops.{
-    |  PipeableImplicit,
-    |  FilterMapExtImplicit,
-    |  FilterMapArraysImplicit,
-    |  FilterMapIteratorsImplicit,
-    |  FilterMapGeneratorsImplicit,
-    |  SeqFactoryFunc,
-    |  RegexContextMaker,
-    |  Callable1Implicit
-    |}
-    |import ammonite.runtime.tools._
-    |import ammonite.repl.tools._
-    |import ammonite.interp.api.IvyConstructor.{ArtifactIdExt, GroupIdExt}
-    |import ammonite.interp.api.InterpBridge.value.exit
-    |""".stripMargin
+  val predefImports = Imports(
+    ImportData("ammonite.interp.api.InterpBridge.value.exit"),
+    ImportData("ammonite.interp.api.IvyConstructor.{ArtifactIdExt, GroupIdExt}", importType = ImportData.Type),
+    ImportData("""ammonite.ops.{
+      PipeableImplicit,
+      FilterMapExtImplicit,
+      FilterMapArraysImplicit,
+      FilterMapIteratorsImplicit,
+      FilterMapGeneratorsImplicit,
+      SeqFactoryFunc,
+      RegexContextMaker,
+      Callable1Implicit
+    }"""),
+    ImportData("ammonite.runtime.tools.{browse, grep, time}"),
+    ImportData("ammonite.runtime.tools.{GrepResult, Grepper, tail}", importType = ImportData.TermType),
```

Comment:
Maybe `GrepResult` and `Grepper` aren't actually needed here…

Reply:
Let's remove them!

Reply:
Yes, confirmed locally they aren't needed.